README.md

```diff
@@ -121,6 +121,10 @@ configure them as something other than the defaults.
 | `ECS_CONTAINER_STOP_TIMEOUT` | 10m | Time to wait for the container to exit normally before being forcibly killed. | 30s |
 | `ECS_ENABLE_TASK_IAM_ROLE` | `true` | Whether to enable IAM Roles for Tasks on the Container Instance | `false` |
 | `ECS_ENABLE_TASK_IAM_ROLE_NETWORK_HOST` | `true` | Whether to enable IAM Roles for Tasks when launched with `host` network mode on the Container Instance | `false` |
+| `ECS_DISABLE_IMAGE_CLEANUP` | `true` | Whether to disable automated image cleanup for the ECS Agent. | `false` |
```

Comment:
@gregmarr Unfortunately there's a bit of technical debt in our `config` package that makes having boolean settings with a default value of `true` impossible to override to `false`.  Until we address that technical debt, all boolean settings need to have a default value of `false`, which means that ones involving features we want on by default end up being negative options.